### PR TITLE
Updated to support Xcode 11.0 beta 4

### PR DESCRIPTION
**Pull request by JS1010111**
Minor changes to compile with Xcode 11.0 beta 4 without errors/warnings.

### CODE REVIEW

File: Sources/SwURL/RemoteImage/RemoteImage.swift [hunk 1]
```diff
@@ -13,7 +13,7 @@ import Combine
 @available(iOS 13.0, *)
 class RemoteImage: BindableObject {
 
-    var didChange = PassthroughSubject<Image?, Never>()
+    var willChange = PassthroughSubject<Image?, Never>()
```
Comment (asos-callumtrounce) [inline Sources/SwURL/RemoteImage/RemoteImage.swift]: @JS1010111 is this simply just a different syntax / naming?  have you checked for behaviour changes?

**Comment by asos-callumtrounce** [comment]
@JS1010111 lets get in touch over slack or discord. what's your slack / discord username?

**Comment by cmtrounce** [comment]
@JS1010111 
what's the best way to get in touch with you? slack or discord?

**Comment by JS1010111** [comment]
Hi @cmtrounce ,
This PR was good for beta 4, I tested it on a project and it worked without any issues. However, it probably has warnings on beta 5 released yesterday (I didn't have the chance to check it yet).

The didChange was deprecated and we should call willChange on willSet (before actually making the changes).
Apple has some new diffing APIs and they say this way things are faster.

In beta 5 there are protocol changes and a new property wrapper `@Published`:

> ObservableObject automatically synthesizes objectWillChange and emits before any @Published properties change.

Take a look at SwiftUI changes on the beta 5 release notes:
https://developer.apple.com/documentation/ios_ipados_release_notes/ios_ipados_13_beta_5_release_notes

Sorry I'm not currently using Slack or Discord.

**Comment by cmtrounce** [comment]
Hi @JS1010111 ,

just to let you know, im in the process of setting up Travis CI so that pull requests, and branches can be tested / built automatically. 

If it will allow you to, could you please branch from my repository rather than using your own fork? 

Many Thanks,

Kind Regards

**Pull request merged**